### ISSUE TITLE
SvgText fill missing when custom attribute style set

### ISSUE THREAD

**Issue by andrewdavey** [vvvv/SVG#507]
Using Svg from nuget version 2.4.3, I'm creating text that has a fill color, and also a custom style attribute set. Back in v1.7 this used to work, however since upgrading, the element is serialized without an explicit `fill` attribute. My guess is the library serializes the fill as a `style`, but my custom style overwrites that.

```
new SvgText("Test")
{
    Fill = blueColor,
    CustomAttributes = { { "style", "cursor: pointer" } }
}
```

I'd expect this to give me

```
<text fill="blue" style="cursor: pointer">Test</text>
```

But instead I get:

```
<text style="cursor: pointer">Test</text>
```

Is this use of the style attribute to set fill, instead of outputting an attribute, a deliberate change in the API?

**Comment by mrbean-bremen** [comment]
As far as I can see, this has been introduced in [this commit](https://github.com/vvvv/SVG/commit/7ece1215b6b77e0469cc936c6284cedc18de146e). Not sure if @erdomke is around to discuss this, but it looks it is there for a reason... And it was introduced not long after the 1.7 release.

**Comment by erdomke** [comment]
Unfortunately, I don't remember the reason for that change.

**Comment by mrbean-bremen** [comment]
> Unfortunately, I don't remember the reason for that change.

Ah well, this was 5 years ago, so I didn't really expect this - I have trouble to remember stuff that was two weeks ago 😛 . Anyway, thanks!

**Comment by mrbean-bremen** [comment]
I may have a closer look at this at another time, or maybe @H1Gdev has some ideas, he seems to know the related code well enough!

**Comment by H1Gdev** [comment]
I do not know process of collecting in `style` attribute.

However, looking at implementation, hopes to be 
`<text style="fill:blue;cursor: pointer">Test</text>`

**Comment by H1Gdev** [comment]
@andrewdavey

Fix in PR #513.

But if it is an addition to `style`, I recommend using `AddStyle` method instead of `CustomAttributes` property.

```cs
var doc = new SvgDocument
{
    Width = 100,
    Height = 100,
};
var text = new SvgText("Test")
{
    X = new SvgUnitCollection(),
    Y = new SvgUnitCollection(),
    Fill = new SvgColourServer(System.Drawing.Color.Blue),
};
text.X.Add(new SvgUnit(0f));
text.Y.Add(new SvgUnit(20f));
text.AddStyle("cursor", "pointer", 0);
doc.Children.Add(text);
doc.Write("output.svg");
```
